src/modules/dashboard/sagas/utils/uploadIfpsAnnotation.ts

```diff
@@ -0,0 +1,28 @@
+import { call } from 'redux-saga/effects';
+import { ipfsUpload } from '../../../core/sagas/ipfs';
+
+export function* uploadIfpsAnnotation(annotationMessage: string) {
+  let ipfsHash: string | null = null;
```

Comment:
Nice work, bringing it all in here, much cleaner.

Is it worth wrapping in a try/catch to add further redundancy here?

Reply:
Not really, ipfs just fails without an error so it won't do anything.